bee-network/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Networking functionality and types for nodes and clients participating in the IOTA protocol built on top of
+//! `libp2p`.
+
+#![deny(missing_docs, warnings)]
+
+mod peer;
+
+// Exports
+pub use self::peer::{PeerInfo, PeerRelation};
+
+// Re-Exports
+#[doc(inline)]
+pub use libp2p::{
+    core::identity::{ed25519::Keypair, PublicKey},
+    multiaddr::Protocol,
+    Multiaddr, PeerId,
+};
+
+/// Creates a (shorter) peer alias from a peer id.
+#[macro_export]
+macro_rules! alias {
```

Comment:
Can this not be a function?

Reply:
It could ... of course, but in this particular case I decided against it to free the user from having to care about whether s/he works with an owned or borrowed peer id.

Reply:
Further context: it's often used in debugging scenarios: 
`debug!("Peer {} didn't play by the rulez!", alias!(very_bad_peer))`.